### PR TITLE
Add CodeJam library

### PR DESCRIPTION
**Pull request by NN---**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,6 +151,15 @@ This library will shape-shift your forms experience in ASP.NET MVC.
 [chameleonforms-gh]:    https://github.com/MRCollective/ChameleonForms
 [chameleonforms-nuget]: https://www.nuget.org/packages/ChameleonForms
 
+## CodeJam
+
+<sup>[GitHub][codejam-gh][/sup> <sup>[Nuget][codejam-nuget]</sup>
```
Comment (atifaziz) [inline README.md:156]: The closing tag of the first `<sup>` tag has a typo.

Comment (NN---) [reply README.md:156]: :+1:

Comment (NN---) [reply README.md:156]: :+1:

Comment (tallesl) [reply README.md:156]: Good catch, fixed.

**Pull request merged**